db/import_pindex.go

```diff
@@ -75,40 +75,37 @@ func getNewPIndexImplType(ctx context.Context) func(indexType, indexParams, path
 	newImportPIndexImpl := func(indexType, indexParams, path string, restart func()) (cbgt.PIndexImpl, cbgt.Dest, error) {
 		defer base.FatalPanicHandler()
 
-		importDest, err := getListenerImportDest(ctx, indexParams)
+		importDest, err := getListenerImportDest(ctx, indexParams, restart)
 		if err != nil {
 			base.ErrorfCtx(ctx, "Error creating NewImportDest during NewImportPIndexImpl: %v", err)
 		}
 		return nil, importDest, err
 	}
 	return newImportPIndexImpl
 }
-func OpenImportPIndexImpl(indexType, path string, restart func()) (cbgt.PIndexImpl, cbgt.Dest, error) {
+
+// openImportPIndexImpl is required to have an implementation from cbgt.PIndexImplType.Open. When this function fails, PIndexImplType will fall back to using PIndexImplType.OpenUsing
```

Comment:
Is this correct?  (that Open falls back to OpenUsing). I don't see that happening in the current cbgt - I don't think cbgt uses Open at all any more (OpenPIndexImpl is only called by tests)

Reply:
I would argue this is a bug in cbgt, but it is required by https://github.com/couchbaselabs/cbgt/blob/master/manager_planner.go#L294

It should check `OpenUsing` instead of `Open` here.